### PR TITLE
Change all images for UG

### LINKED ISSUE #106: [PE-D][Tester B] Unclear visuals in addavail (and possibly other commands)
Visuals may be difficult to see the changes, I'd recommend adding circles to show what changed. 

i.e. in addavail "what you should see" diagram, would be good to circle the dates that were added.
Alternatively, could show a before and after.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: blaukc/ped#4

### PR DESCRIPTION
**Pull request by xuwen-ho**
fix #106 

Change the images of various commands in Match

Previously, some images contained inaccuracies or were misaligned

Lets make sure images are accurate

### CODE REVIEW

File: docs/images/features/email.png
Comment (SimKianSeng) [inline docs/images/features/email.png:1]: perhaps can add a bullet point to indicate what is this about? For a moment I was confused when I saw this.

**Review by tayziyi — CHANGES_REQUESTED** [review]
Overall good job! Possible to help fix issue #106 as well since it is linked to visuals? Thanks!

**Review by SimKianSeng — COMMENTED** [review]
Minor clarity issue

**Review by Benson15912 — APPROVED** [review]
lgtm

**Review by tayziyi — APPROVED** [review]
lgtm

**Pull request merged**